app/components/widget-adapter/component.js

```diff
@@ -1,6 +1,8 @@
 import Ember from 'ember';
 import ENV from 'analytics-dashboard/config/environment';
 import stringify from 'npm:json-stable-stringify';
+import tsInterval from '../../utils/date-interval';
```

Comment:
This should be `import dateInterval from '../../utils/date-interval';` since the name changed